public/app/plugins/datasource/prometheus/configuration/ExemplarsSettings.tsx

```diff
@@ -0,0 +1,94 @@
+import { DataSourcePluginOptionsEditorProps, updateDatasourcePluginJsonDataOption } from '@grafana/data';
+import { InlineField, Input, Switch, useStyles } from '@grafana/ui';
+import { DataSourcePicker } from 'app/core/components/Select/DataSourcePicker';
+import { css } from 'emotion';
+import React, { useState } from 'react';
+import { PromOptions } from '../types';
+
+interface Props extends DataSourcePluginOptionsEditorProps<PromOptions> {}
+
+export function ExemplarsSettings({ options, onOptionsChange }: Props) {
+  const [isInternalLink, setIsInternalLink] = useState(
+    Boolean(options.jsonData.exemplarTraceIDDestination?.datasourceUid)
+  );
+  const styles = useStyles(getExemplarsSettingsStyles);
+
+  return (
+    <>
+      <h3 className="page-heading">Exemplars</h3>
+
+      <InlineField label="Internal link" labelWidth={24}>
+        <div className={styles.switch}>
+          <Switch value={isInternalLink} onChange={ev => setIsInternalLink(ev.currentTarget.checked)} />
```

Comment:
The styling of this toggle is different than rest of the toggles. Border is missing.
![image](https://user-images.githubusercontent.com/30407135/104312726-4b1fa900-54d7-11eb-8aa2-c5729fe592cb.png)

Reply:
Good catch! i think @torkelo added the component in https://github.com/grafana/grafana/pull/30162

Reply:
yep